rise-playlist.html

```diff
@@ -176,8 +168,8 @@
           duration = parseInt(duration, 10);
 
           if (!isNaN(duration)) {
-            setTimeout(function() {
-              self._playNext(index, "pause");
+            this.debounce("play-next", function() {
+              this._playNext(index, "pause");
```

Comment:
@donnapep Not so much an advantage I guess, just a simpler execution - `async` will call your method as soon as the delay that is applied is finished. You're right though about `debounce` preventing more than one call, it is good to leverage.